src/main/java/org/truffleruby/core/kernel/KernelNodes.java

```diff
@@ -1193,31 +1193,9 @@ protected boolean isATypeError(Object self, Object module) {
 
         @Child private WarnNode warnNode;
```

Comment:
This and `warnProcWithoutBlock()` below can be removed too.

Reply:
I forgot to push `$ jt checkstyle` result 🙈 Fixed

Reply:
I still see the field above in the diff, could you remove it?

Reply:
Meh, this one I forgot to remove; Sorry, I should be a bit more attentive. Done